sw/airborne/modules/nav/nav_smooth.c

```diff
@@ -194,8 +194,8 @@ bool snav_on_time(float nominal_radius)
   /* Recompute ground speeds every 10 s */
   if (ground_speed_timer == 0) {
     ground_speed_timer = 40; /* every 10s, called at 40Hz */
-    compute_ground_speed(airspeed, stateGetHorizontalWindspeed_f()->y,
-                         stateGetHorizontalWindspeed_f()->x); // Wind in NED frame
+    compute_ground_speed(airspeed, stateGetHorizontalWindspeed_f()->x,
```

Comment:
still wondering about this one... think it is actually correct, although variable naming definitely leaves to be desired...
Seems wind_x is expected to be east and wind_y north (so ENU), but stateGetHorizontalWindspeed_f returns it in NED